README.md

```diff
@@ -18,13 +18,13 @@ To install the command line tool globally, run:
 
 ## Run using Docker
 
-Build a Docker image:
-
-    docker build --tag markdown-link-check .
+Docker images are built with each release. Use the `stable` tag for the current stable release.
 
 Pipe your `README.md` file to `docker run`:
 
-    docker run --rm -i markdown-link-check < README.md
+    docker run --rm -i ghcr.io/tcort/markdown-link-check:stable < README.md
```

Comment:
As the tcort image is not published yet, I tried `docker run --rm -i ghcr.io/cadeef/markdown-link-check < README.md`. But we have an issue on the image that returns:
```
internal/fs/utils.js:612
    throw new ERR_INVALID_ARG_TYPE(propName, ['string', 'Buffer', 'URL'], path);
    ^

TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string or an instance of Buffer or URL. Received undefined
    at Object.stat (fs.js:1014:10)
    at Command.<anonymous> (/src/markdown-link-check:65:12)
    at Command.listener [as _actionHandler] (/src/node_modules/commander/index.js:426:31)
    at Command._parseCommand (/src/node_modules/commander/index.js:1002:14)
    at Command.parse (/src/node_modules/commander/index.js:801:10)
    at Object.<anonymous> (/src/markdown-link-check:75:4)
    at Module._compile (internal/modules/cjs/loader.js:1076:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1097:10)
    at Module.load (internal/modules/cjs/loader.js:941:32)
    at Function.Module._load (internal/modules/cjs/loader.js:782:14) {
  code: 'ERR_INVALID_ARG_TYPE'
}
```
So two comments comes to my mind:

- should we fix it before publishing the image? it's #67 
- should we implement some tests on the image once built before publishing?

Reply:
#67 should be fixed regardless of the PR. It appears that receiving content on STDIN has been broken for quite some time. I, in fact, stumbled on the examples when I first looked at markdown-link-check. The examples in the README that use STDIN all fail.

It may be practical to update the README with file-based examples until it's fixed. The Docker example, specifically, would need to be modified to include a bind mount so it could work on files directly.

As for integration testing on the image, I agree it's needed. I tend to scope PRs to the issue where possible.

I'd be willing to implement basic [bats-based](https://github.com/bats-core/bats-core) testing in another PR.

A larger conversation about acceptance testing for the entire repository should probably be had. Should I create an issue to kick off said conversation?

Reply:
Of course #67 will have to be fixed. My point was to make sure we were ok to push an image with this issue until it's solved.
Also, splitting PRs in small things is the way to go.
For the testing discussion, yes please create a ticket.
thanks for your help!